src/commander.c

```diff
@@ -320,7 +320,7 @@ static int command_brakes( )
                 normalized_position,
                 BRAKE_FILTER_FACTOR );
 
-            printf("Brake:\t%f\n", average);
+            printf("brake: %f ", average);
```

Comment:
I'm seeing both capitalized and lower-case error messages, for example in this PR:
https://github.com/PolySync/oscc-joystick-commander/pull/12/files

Reply:
The original code base was lowercase print statements but we are not switching to capitalized print statements.

9d7e22c